docs/index.rst

```diff
@@ -21,4 +21,3 @@ Sound interesting? Read on!
    README
    usage
    frontend-protocol
-   protocol
```

Comment:
So this piece of the protocol _is_ actually used internally (still) but I guess this doc doesn't make that part very clear (i.e. needs an "audience: developers of fowl" or something tag).

This is the protocol we speak over dilation to each fowl process